python/lsst/summit/utils/nightReport.py

```diff
@@ -762,7 +762,7 @@ def makeAltAzCoveragePlot(
         ax.set_title("Axial coverage - azimuth (theta, deg) vs zenith angle (r, deg)", size=20)
 
         for h in lgnd.legend_handles:
-            if h is None:
+            if h is None or not isinstance(h, matplotlib.lines.Line2D):
```

Comment:
Kind of gross, but I guess this is because matplotlib hasn't properly annotated the return type above? Or it has, and it can return other things?

Reply:
It can return other things...